src/pages/App.jsx

```diff
@@ -18,6 +18,7 @@ export const App = () => {
           <Route path="/" element={<Landing />} />
           <Route path="/about" element={<About />} />
           <Route path="/categories" element={<Categories />} />
+          <Route path="/walletviewpage/:id" element={<WalletViewPage />} />
```

Comment:
```suggestion
          <Route path="/wallet/:id" element={<WalletViewPage />} />
```
try to make your paths as short as possible

Reply:
done